app/views/job_templates/rh_cloud_download_playbook.erb

```diff
@@ -0,0 +1,14 @@
+<%#
+kind: job_template
+name: Run a playbook downloaded from RH cloud
+job_category: Red Hat Insights
+description_format: 'Playbook generated by RH Cloud'
+feature: rh_cloud_connector_run_playbook
+template_inputs:
+- name: playbook_url
+  description: URL of the playbook to run
+  input_type: user
+  required: true
+provider_type: Ansible
+%>
+<%= download_rh_playbook(input('playbook_url')) %>
```

Comment:
When and where does this mean the download of the playbook happens?

Reply:
the `download_rh_playbook` method will do the downloading once invoked.